pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -52,6 +52,10 @@ var (
 	errNoActiveClusters = errors.New("no active clusters")
 )
 
+const (
+	MultiKueueOriginLabelKey = "kueue.x-k8s.io/multikueue-origin"
```

Comment:
Do we have some dedicated package or file for consts? It would be nice to group it together with other labels / annotations, for code organization purposes.

Reply:
there are a couple `pkg/constants` ,`pkg/controller/constants`  but I don't see this fitting in any, being very speiffic to multikueue.

We could add a cleanup follow-up to group the multikueue constants in a dedicate file in this package.

Reply:
I see, I would like to have then an analogous constants package for multikueue. Can be a follow up if we have already some other API-facing consts from other PRs

Reply:
Similarly, in the core k8s the API-facing labels are put together: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/batch/types.go#L25-L54. I think this helps a lot with readability to have one go-to place rather than scattered.

Reply:
moved in API

Reply:
sgtm